inkscape/svg2shenzhen/export.py

```diff
@@ -628,32 +628,31 @@ def exportEdgeCut(self, kicad_mod=False):
 
             layer_trans = layer.get('transform')
             if layer_trans:
-                layer_m = simpletransform.parseTransform(layer_trans)
+                layer_m = Transform(layer_trans).matrix
             else:
                 layer_m = IDENTITY_MATRIX
 
             nodePath = ('//svg:g[@inkscape:groupmode="layer"][%d]/descendant::svg:path') % i
             for node in self.document.getroot().xpath(nodePath, namespaces=inkex.NSS):
                 d = node.get('d')
-                p = simplepath.parsePath(d)
+                p = inkex.Path(d).to_arrays()
 
                 points = []
                 if p:
                     #sanity check
                     if p[0][0] == 'M':
                         t = node.get('transform')
                         if t:
-                            m = simpletransform.parseTransform(t)
-                            trans = simpletransform.composeTransform(layer_m, m)
+                            m = Transform(t).matrix
+                            trans = (Transform(layer_m)*Transform(m)).matrix
```

Comment:
There's no reason to do `Transform(Transform(t).matrix))` You can just change `m` to `t` and remove line 646

Reply:
Well spotted. Done.